js/components/ColumnControls.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+/* ColumnControls - populates the cell below the shelfmark label for
+ * each manuscript column in the scriptchart. The cell currently
+ * contains a book icon that opens the manuscript in the Mirador
+ * viewer, and an X that hides the column when clicked, adding
+ * it to the ChartAccordion list.
+ */
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+class ColumnControls extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.showManifest = this.showManifest.bind(this);
+    this.hideColumn = this.hideColumn.bind(this);
+  }
+
+  showManifest() {
+    this.props.displayManifest(
+      this.props.manifestURL,
+      this.props.onManifestSelected
+    );
+  }
+  hideColumn() {
+    this.props.onHiddenChange("hide", "column", this.props.msid);
+  }
+
+  render() {
+    let manifestSpan = "";
+    if (this.props.manifestURL != null) {
+      manifestSpan = (
+        <span style={{ float: "right" }}>
+          <FontAwesomeIcon
+            title="Show this manuscript in the Mirador viewer"
+            color="#0000FF"
+            icon="book-open"
+            onClick={this.showManifest}
+            style={{ cursor: "pointer" }}
+          />
+        </span>
+      );
+    }
+    return (
+      <p>
+        <span
+          style={{ float: "left" }}
```

Comment:
I am noticing that styles are sort of scattered between the React components and the sass sheets that Jekyll builds. This is probably fine, but we should probably be able to articulate why different styles go in different places.

Reply:
Good point. Actually I just realized that the FontAwesome icon style in ColumnControls.js is exactly the same as in DashTabs.css (imported in DashTabs.js), so there's one easy opportunity to begin consolidating styles.
Coordinating between Jekyll and React will be an ongoing effort, I'm sure. It also would be nice to find a way to reduce the footprint of the FontAwesome elements.

Reply:
Agreed. I think we can try to keep any styles that affect parts of the Jekyll templates in the sass files that Jekyll builds, and anything affecting the React components, as much as possible, in css files that React imports. We won't worry about the the fontawesome stuff in this PR, but it would be good to note it in an issue and see if we can resolve it.

Reply:
Issue re: the fontawesome stuff
https://github.com/sul-cidr/scriptchart/issues/39